tests/test_core.py

```diff
@@ -8570,6 +8570,16 @@ def test():
     self.emcc_args += ['-DPOOL']
     test()
 
+  @parameterized({
+    'dlmalloc': ['dlmalloc'],
+    'emmalloc': ['emmalloc'],
+  })
+  def test_malloc_alignment(self, malloc):
+    self.set_setting('MALLOC', malloc)
+    self.emcc_args += ['-std=gnu11']
+    src = open(path_from_root('tests', 'core', 'test_malloc_alignment.c')).read()
+    self.do_run(src, '', basename='src.c')
```

Comment:
These days you can just write `self.do_runf(path_from_root('tests', 'core', 'test_malloc_alignment.c'))`